src/Test/L0/Worker/ActionCommandManagerL0.cs

```diff
@@ -418,6 +418,24 @@ public void AddMatcherTranslatesFilePath()
             }
         }
 
+        [Fact]
+        [Trait("Level", "L0")]
+        [Trait("Category", "Worker")]
+        public void AddMaskWithMultilineValue()
+        {
+            using (TestHostContext hc = CreateTestContext())
+            {
+                // Act
+                _commandManager.TryProcessCommand(_ec.Object, $"::add-mask::abc%0Ddef%0Aghi", null);
```

Comment:
consider adding the crlf example that windows typically uses `\r\n` 
`abc\rdef\nghi\r\njkl`